docker/Dockerfile.agent.alpine.multiarch

```diff
@@ -0,0 +1,18 @@
+FROM --platform=$BUILDPLATFORM golang:1.16 AS build
+
+WORKDIR /src
+COPY . .
```

Comment:
Should we add individual dockerignore files?

Reply:
What are you going to ignore?

Reply:
I would use something like the following dockerignore for the agent. This would for example exclude the web and docs folders. 

```
*
!go.mod
!go.sum
!vendor/modules.txt
!*.go
!Makefile
```

Reply:
@anbraten done

see https://github.com/moby/buildkit/pull/901

(for manual testing: enable buildkit mode: `export DOCKER_BUILDKIT=1`)

Reply:
uh wait - I think i have to fix the igore file

Reply:
@anbraten looks like we need the whole repo in the first build context - the resulting container will NOT contain that anyway ....